README.md

```diff
@@ -52,3 +52,44 @@ There are three steps to the CI workflow:
      condition that a service must have been updated to run. The `deploy` job
      doesn't update any services, only manifests, so the push from the `deploy`
      job will never run another `deploy` job.
+
+## WIP: deployment to kubernetes
+
+Create a kubernetes cluster somewhere
+
+```bash
+gcloud container clusters create dapr --num-nodes=1
+```
+
+Install Dapr on the cluster
+
+```bash
+dapr init -k
+```
```

Comment:
We should probably leave a link to https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/

Reply:
I've added this